packages/commons/src/pdf-generator/pdfGenerator.ts

```diff
@@ -93,5 +94,60 @@ export async function initPDFGenerator(): Promise<PDFGenerator> {
         await page?.close();
       }
     },
+    screenshotFromBytes: async (pdfContent: Uint8Array): Promise<Buffer> => {
```

Comment:
Not needed I think

Reply:
removed in commit [1601718](https://github.com/pagopa/interop-be-monorepo/pull/1214/commits/1601718dec854f0ea2dab0aa3eedb308edacf96f)